backend/config.py

```
@@ -0,0 +1,276 @@
+"""
+TransDock Configuration Module
+
+Centralizes all environment variable loading and configuration management
+for improved code organization and maintainability.
+"""
+
+import os
+import secrets
+import string
+from pathlib import Path
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)